shared/constants/platform.d.ts

```diff
@@ -23,4 +23,5 @@ export declare const version: string
 export declare const pprofDir: string
 export declare const serverConfigFileName: string
 export declare const shortcutSymbol: string
+export declare const deviceName: string
```

Comment:
I find this name a little confusing because it's the same thing that we use for the Keybase device name. Maybe like `realDeviceName` or something?

Reply:
or perhaps `defaultDeviceName`?

Reply:
I like `realDeviceName`!